docs/_static/css/pyodide.css

```diff
@@ -6,3 +6,14 @@
 code {
     font-size: 100% !important;
 }
+
+
+h3 {
```

Comment:
Maybe let's add another sector specific to the page content. Never know where h3 / h4 could be used.

Reply:
Well here again I looked at the selector that was determining the size of the headers and it's just an `h3` by itself. But it seems that I can do `.section > h3` which is probably better.